test/test_dispatch.py

```diff
@@ -374,6 +374,114 @@ def test_computed_table_with_cpu_catchall(self):
 AutogradCPU: fallthrough registered at /dev/null:0 [backend fallback]
 AutogradCUDA: default_def_name_t_t [catch all]
 AutogradXLA: default_def_name_t_t [catch all]
+''')
```

Comment:
Haven't looked closely at these, but just tossing in a repeat of the comment about Autograd vs Math from above